src/fragments/lib/geo/js/search.mdx

```diff
@@ -177,3 +177,101 @@ const searchOptionsWithBiasPosition = {
 
 Geo.searchByCoordinates([-122.3399573, 47.616179], searchOptionsWithBiasPosition)
 ```
+
+### Search for suggestions
+
+The `Geo.searchForSuggestions()` API enables you to search for suggestions by free-form text, such as a place, address, city, or region.
+
+```javascript
+import { Geo } from "aws-amplify"
+
+Geo.searchForSuggestions("Amazon Go Store")
+```
+
+Similar to `Geo.searchByText()` API, customize your search results further by providing:
+- `countries` - to limit the search results to given countries (specified in [ISO Alpha-3 country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3))
+- `maxResults` - to limit the maximum result set
+- `biasPosition` - to act as the search origination location
+- `searchAreaConstraints` - to limit the area to search inside of
+- `searchIndexName` - to use a different Location Service search index resource than the default
+
+ **Note:** Providing both `biasPosition` and `searchAreaConstraints` parameters simultaneously returns an error.
+
+```javascript
+const searchOptionsWithBiasPosition = {
+  countries: string[], // Alpha-3 country codes
+  maxResults: number, // 50 is the max and the default
+  biasPosition: [
+    longitude // number
+    latitude // number,
+  ], // Coordinates point to act as the center of the search
+  searchIndexName: string, // the string name of the search index
+}
+
+const searchOptionsWithSearchAreaConstraints = {
+  countries: ["USA"], // Alpha-3 country codes
+  maxResults: 25, // 50 is the max and the default
+  searchAreaConstraints: [SWLongitude, SWLatitude, NELongitude, NELatitude], // Bounding box to search inside of
+  searchIndexName: string, // the string name of the search index
+}
+
+Geo.searchForSuggestions('Amazon Go', searchOptionsWithBiasPosition)
+```
+This returns a list of suggestions (places and their respective `placeId` if available) that match the search constraints.
+
+```javascript
+// returns
+[
+  {
+    text: "Amazon Go, 2131 7th Ave, Seattle, WA, 98121, USA",
+    placeId: "8fd9d4c6-2527-4190-a7df-0dae352c9dc6"
+  },
+  {
+    text: "Amazon Go, 1906 Terry Ave, Seattle, WA, 98101, USA",
+    placeId: "5d04d071-dea2-4d86-bfce-86bd6a8f4787"
+  }
+]
```

Comment:
Can you include an example here of result without a `placeId` and advice that they can use that result with the `searchByText` API. Otherwise LGTM

Reply:
![SCR-20221121-e42](https://user-images.githubusercontent.com/12038441/203129062-936cfba1-7baa-4c04-a202-59ca56cba50e.png)